Makefile

```diff
@@ -7,12 +7,17 @@ OCAML_LIB_DIR=$(shell ocamlc -where)
 
 CTYPES_LIB_DIR=$(shell ocamlfind query ctypes)
 
-OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) OCAML_LIB_DIR=$(OCAML_LIB_DIR) \
-	ocamlbuild -use-ocamlfind -classic-display
+LWT_LIB_DIR=$(shell ocamlfind query lwt)
+
+OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) \
+           LWT_LIB_DIR=$(LWT_LIB_DIR)       \
+           OCAML_LIB_DIR=$(OCAML_LIB_DIR)   \
+             ocamlbuild -use-ocamlfind -classic-display
 
 WITH_UNIX=$(shell ocamlfind query \
             ctypes unix unix-type-representations unix-errno.unix \
             > /dev/null 2>&1 ; echo $$?)
+WITH_LWT=$(shell ocamlfind query lwt unix > /dev/null 2>&1 ; echo $$?)
```

Comment:
I agree, it's horrible (and I've just made it more magical in a recent commit). I don't really want to create more opam packages to represent these features... need feature support in opam but I don't know the status of ocaml/opam#1985.